src/integ_test_resources/android/amplify/integration/cdk/scripts/api.py

```diff
@@ -0,0 +1,88 @@
+import json
+import os
+from common import *
+
+class ApiConfigBuilder:
+    name = None
+    schema_dir = None
+    default_auth_mode = None
+    additional_auth_modes = {}
+    is_update = False
+
+    def as_update(self):
+        self.is_update = True
+
+    def with_name(self, name: str):
```

Comment:
Since Python includes support for named arguments, you usually see the `__init__` method used directly instead of these builder-style methods. (In Java, we use Builders a lot, not because we do elaborate configurations, but to "patch" in named arguments.)

Reply:
ahh...that makes sense. I can change that. Old habits die hard.

Reply:
actually, I'll make this change in a separate PR.